src/ubx.cpp

```diff
@@ -83,9 +83,7 @@ GPSDriverUBX::GPSDriverUBX(Interface gpsInterface, GPSCallbackPtr callback, void
 
 GPSDriverUBX::~GPSDriverUBX()
 {
-	if (_rtcm_parsing) {
-		delete (_rtcm_parsing);
-	}
+	delete _rtcm_parsing;
```

Comment:
@bkueng it looks like the bug. `_rtcm_parsing` is not created in simple GPS OutputMode. It is not a good idea to remove `_rtcm_parsing` null checking.

Reply:
Passing `nullptr` to `delete` is valid

Reply:
Yeah, you are right, I missed it. Then it also could be done in other drivers, too :)

Reply:
Yes, definitely